components/PerangkatDesa/index.vue

```diff
@@ -44,10 +44,76 @@
           <PerangkatDesaTableAction
             :status="item.status"
             @delete="deleteData(item)"
+            @verify="verifyUser(item)"
           />
         </template>
       </BaseDataTable>
     </div>
+    <BaseCustomDialog
+      :show="showDialogVerify"
+      :header="contentPerangkatDesa.header"
+      :title="contentPerangkatDesa.title"
+      :data="contentPerangkatDesa.data"
+    >
+      <div v-if="contentPerangkatDesa.showNotes === true" class="mx-6 mb-6">
```

Comment:
minor/discussion:

Hello @yoslie , I just wondering what is the data type of `contentPerangkatDesa.showNotes` ? If it's Boolean I think we just need to write `v-if="contentPerangkatDesa.showNotes"` or `v-if="!contentPerangkatDesa.showNotes`

Reply:
yes it is boolean, thanks changes [here](https://github.com/jabardigitalservice/dedi-cms/pull/82/commits/90253b9f98841c5d582fb4fd2b59c18488ce7b3f)